frontend/src/constants/apy-mainnet-constants.ts

```diff
@@ -36,5 +36,14 @@ export const COINGECKO_TOKEN_NAME_TRANSLATION: {[key: string]: string} = {
     "eur": "eur"
 }
 
-export const validCurveTokens = Object.keys(CURVE_SWAP_ADDRESSES);
+
+export const validCurveTokens = [
+    "crv3crypto",
+    "crvtricrypto",
+    "stecrv",
+    "lusd3crv-f",
+    "alusd3crv-f",
+    "mim-3lp3crv-f",
+    "eurscrv",
```

Comment:
does this list needs to be updated when element deploys a new crv like token? If so can you create an appropriate TODO?

Reply:
It does. I'll add one.